protocol/message.go

```diff
@@ -106,6 +181,14 @@ func NewKeyLookupProof(ap *m.AuthenticationPath, str *m.SignedTreeRoot,
 	}, e
 }
 
+// NewKeyLookupInEpochProof creates the response message a CONIKS server
+// sends to a client upon a KeyLookupInEpochRequest.
+// The contained DirectoryProofs is either a single proof of inclusion
+// (e = Success), or a proof of absence if the requested name does not
+// exist in the directory (e = ErrorNameNotFound).
+// In either case, the server returns a list of STRs for the epoch range [ep,
+// directory.LatestEpoch().Epoch], where ep is the past epoch for which
+// the client has requested the user's key.
```

Comment:
I'm a bit confused, what do you mean by "behavior of the server"? Are you suggesting `KeyLookupInEpoch` shouldn't generate the list of STRs but that this should be done in the server? The list of STRs *is* part of the `KeyLookupInEpoch` protocol, though. Maybe my wording and use of the term "server" here is misleading?